build.gradle.kts

```diff
@@ -48,7 +48,7 @@ gradlePlugin {
 object PluginCoordinates {
 
     const val GROUP = "com.wire"
-    const val ARTIFACT = "carthage-plugin"
+    const val ARTIFACT = "carthage-gradle-plugin"
```

Comment:
Might need to to update something in the Kalium branch now that this changed.

Reply:
That's done